packages/help-panel/src/components/HelpPanel/Containers/FloatingContainer/FloatingContainer.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { forwardRef } from 'react'
+import Container from '../../../Container'
+import css from './FloatingContainer.module.css'
+import floating_button from '../../../../icons/floating_button.svg'
+import { HelpPanelContext } from '../../../../HelpPanelContext'
```

Comment:
can we use root-relative paths (using resolutions)  instead of `../` relative paths?

Reply:
Facing some issues in adding aliases, will pick this up in the follow up PR